.github/workflows/checks.yml

```diff
@@ -0,0 +1,16 @@
+name: Merge check
```

Comment:
Note that Github will use the name of the workflow for the default badge. So you'll get ![Merge check](https://github.com/y-lohse/inkjs/workflows/Merge%20check/badge.svg). It may not be what you want, but it can certainly be worked around by using shield.io.

Reply:
Cool, I updated the names… and the badge that I forgot about.